src/document.h

```diff
@@ -830,17 +844,18 @@ struct Document {
         return nullptr;
     }
 
-    const wxChar* CopyImageToClipboard(Cell *cell) {
+    const wxChar* CopyImageToClipboard(Cell *&cell) {

```

Comment:
please remove this change here and elsewere, you are not even changing the `cell` variable??